common/OpTestUtil.py

```diff
@@ -1116,8 +1116,10 @@ def configure_host_ip(self, interface, ip, netmask, cv_HOST):
             cv_HOST.host_run_command(flush_interface)
             ip_config = "ip addr add {}/{} dev {}".format(ip, netmask, interface)
             cv_HOST.host_run_command(ip_config)
+            return True
         except CommandFailed as cf:
             log.debug("configure ip to interface  CommandFailed={}".format(cf))
+            return False
```

Comment:
True and False should be return based on some condition rite?  meaning true if ip addr add command executed fine

Reply:
flush the interface before configuring ip, so host_run_command wont exit with non zero